src/payment/cart/Cart.jsx

```diff
@@ -56,72 +56,72 @@ class Cart extends React.Component {
         </span>
 
         <CartContents>
-          {products && products.length ? products.map(product => (
+          {products.map(product => (
             <ProductLineItem
               key={product.title}
               isNumEnrolledExperiment={isNumEnrolledExperiment}
               REV1045Experiment={REV1045Experiment}
               isPaymentVisualExperiment={isPaymentVisualExperiment}
               enrollmentCountData={enrollmentCountData}
               {...product}
-            />)) : (REV1045Experiment ?
-              <div className="row align-items-center mb-5">
-                <div className="col-5">
-                  <div className="skeleton embed-responsive embed-responsive-16by9" />
-                </div>
-                <div className="col-7">
-                  <div className="skeleton py-2 mb-3 w-50" />
-                  <div className="skeleton py-2 mr-4" />
-                </div>
-              </div> : null)}
+            />
+          ))}
 
           {isBulkOrder ? <UpdateQuantityForm /> : null}
         </CartContents>
 
-        {!REV1045Experiment || loaded ?
-          <OrderSummary> {isBulkOrder ? (
-            <BulkOrderSummaryTable
-              price={summaryPrice}
-              subtotal={summarySubtotal}
-              quantity={summaryQuantity}
-            />
-          ) : (<SummaryTable price={summaryPrice} />)}
-            <Offers
-              discounts={summaryDiscounts}
-              offers={offers}
-              isBundle={products.length > 1}
-            />
-
-            {showCouponForm ?
-              <CouponForm isPaymentVisualExperiment={isPaymentVisualExperiment} /> : null}
-
-            <TotalTable total={orderTotal} />
-
-            {isCurrencyConverted ? <CurrencyDisclaimer /> : null}
-          </OrderSummary> : (
-            <React.Fragment>
+        {!REV1045Experiment || loaded
```

Comment:
Could also remove all the places where `REV1045Experiment` is in the code - if not on this PR, REV squad will do in a future experiment clean up.